.github/workflows/main.yml

```diff
@@ -0,0 +1,22 @@
+name: smart_content_cdn
+on:
```

Comment:
Suppose we could always change this later, but wondering, do we want to limit this to just one branch?

Reply:
Probably not. That was Copilot being enthusiastic.